### PR TITLE
Snake case fix

### PR DESCRIPTION
**Pull request by PatrikLundgren**
just in snake_case it isnt obvious. Changed variable_names and class_names to snake_case

### CODE REVIEW

File: ViAn/Filehandler/project.h [hunk 1]
```diff
@@ -15,7 +15,7 @@ typedef int ID;
  * Importand for seperation of different types of data.
  */
 struct ProjectStream{
-    std::stringstream projFile;
+    std::stringstream projfile;
```
Comment (svidem) [inline ViAn/Filehandler/project.h]: Shouldn't this be proj_file?

File: ViAn/Filehandler/filehandler.cpp [hunk 2]
```diff
@@ -365,13 +365,13 @@ ID FileHandler::add_dir(std::string dirpath){
  * @param proj2
  * @return true if project contents are the same
  */
-bool FileHandler::proj_equals(Project& proj, Project& proj2){
-    bool videoEquals =  std::equal(proj.m_videos.begin(), proj.m_videos.end(),
-               proj2.m_videos.begin(),
-               [](const Video* v, const Video* v2){return *v == *v2;}); // lambda function comparing using video==
+bool file_handler::proj_equals(project& proj, project& proj2){
+    bool videoEquals =  std::equal(proj.videos.begin(), proj.videos.end(),
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: video_equals

File: ViAn/Filehandler/filehandler.cpp [hunk 3]
```diff
@@ -239,7 +239,7 @@ ID FileHandler::create_file(std::string filename, ID dirID){
   * @param ID file id, std::string text
   * @return void
   */
- void FileHandler::write_file(ID id, std::string text, WRITE_OPTION opt){
+ void file_handler::write_file(ID id, std::string text, WRITE_OPTION opt){
     std::string fileName = this->get_file(id);
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: file_name

File: ViAn/Filehandler/filehandler.cpp [hunk 4]
```diff
@@ -214,9 +214,9 @@ void FileHandler::add_video(Project* proj, std::string filePath){
   * @param std::string file name, ID directory id
   */
 
-ID FileHandler::create_file(std::string filename, ID dirID){
+ID file_handler::create_file(std::string filename, ID dir_id){
     std::ofstream f;
-    std::string filePath = this->get_dir(dirID)+"/"+filename;
+    std::string filePath = this->get_dir(dir_id)+"/"+filename;
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: file_path

File: ViAn/Filehandler/filehandler.cpp [hunk 5]
```diff
@@ -264,7 +264,7 @@ ID FileHandler::create_file(std::string filename, ID dirID){
   *  @param ID file id, std::string text
   *  @return voi
   */
- void FileHandler::read_file(ID id,  std::string& buf, int linesToRead){
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: lines_to_read

File: ViAn/Filehandler/filehandler.cpp [hunk 6]
```diff
@@ -30,7 +30,7 @@ Project* FileHandler::create_project(std::string projName){
  * @param dirpath
  * @return unique directory ID
  */
-ID FileHandler::create_directory(std::string dirpath){
+ID file_handler::create_directory(std::string dirpath){
     this->lastError = make_dir(dirpath); //varying implementation, OS dependant
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: last_error

File: ViAn/Filehandler/filehandler.cpp [hunk 7]
```diff
@@ -3,10 +3,10 @@
...
  */
-FileHandler::FileHandler() {
-    this->m_pid = 0; // zero out counter ids
-    this->m_fid = 0;
-    this->m_did = 0;
+file_handler::file_handler() {
+    this->pid = 0; // zero out counter ids
+    this->fid = 0;
+    this->did = 0;
     this->lastError = 0;
```
Comment (parso619) [inline ViAn/Filehandler/filehandler.cpp]: last_error

File: ViAn/Filehandler/video.cpp [hunk 8]
```diff
@@ -2,7 +2,7 @@
 /**
  * @brief Video::Video
  */
-Video::Video()
+video::video()
 {
```
Comment (parso619) [inline ViAn/Filehandler/video.cpp]: { on new line

File: ViAn/Filehandler/project.h (hunk 1 above)
Comment (PatrikLundgren) [inline ViAn/Filehandler/project.h]: no, this is a middle-hand for reading seperate projectfiles. this one for reading project infofile

**Pull request closed without merging**